protocol/code_review.md

```diff
@@ -0,0 +1,75 @@
+Code Review
+===========
+
+A guide for reviewing code and having your code reviewed.
+
+Everyone
+--------
+
+* Accept that many programming decisions are opinions. Discuss tradeoffs, which
+  you prefer, and reach a resolution quickly.
+* Ask questions; don't make demands. ("What do you think about naming this
+  `:user_id`?")
+* Ask for clarification. ("I didn't understand. Can you clarify?")
+* Avoid selective ownership of code. ("mine", "not mine", "yours")
+* Avoid using terms that could be seen as referring to personal traits. ("dumb",
+  "stupid"). Assume everyone is attractive, intelligent, and well-meaning.
+* Be explicit. Remember people don't always understand your intentions online.
+* Be humble. ("I'm not sure - let's look it up.")
+* Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+* Don't use sarcasm.
+* Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
+  If they are, use them with aplomb.
+* Talk in person if there are too many "I didn't understand" or "Alternative
+  solution:" comments. Post a follow-up comment summarizing offline discussion.
+
+Having Your Code Reviewed
+-------------------------
+
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* Don't take it personally. The review is of the code, not you.
+* Explain why the code exists. ("It's like that because of these reasons. Would
+  it be more clear if I rename this class/file/method/variable?")
+* Extract some changes and refactorings into future tickets/stories.
```

Comment:
I think this could be expanded with a general note on scope: sometimes an issue comes up in code review, and it's not fair to make someone have to fix a project-wide issue to get their PR merged.  So I think a good guideline might be: "Limit code review comments to the code being added or changed.  If the PR raises bigger issues, those should be extracted to their own tickets".